### DISCUSSION TITLE
Disable check for new versions

### DISCUSSION THREAD

**Discussion by jelabarre59** [jellyfin/jellyfin-desktop#670] [General]
On the Linux Jellyfin-media-player, how do I set it to NOT check for updates at startup?  It will want to install a version of the client that requires the Jellyfin 1.9 server, which is not yet available for Fedora.

**Comment by thornbill** [comment]
It requires Jellyfin **10.9** because earlier versions do not work. See this pinned issue: https://github.com/jellyfin/jellyfin-media-player/issues/624

**Reply by jelabarre59** [reply]
So how do I set my current client (jellyfin-media-player-1.9.1-1.fc39.x86_64) to not check for newer versions?  Until Jellyfin Server 1.9 is available on Fedora, I don't want it checking.

**Comment by gallen881** [comment]
On Windows 10, you can find this setting by clicking your avatar in the top right corner and selecting **Client Settings**. Then, uncheck the `CheckForUpdates` box to turn off update checks.

![image](https://github.com/user-attachments/assets/91b1b1e9-51cb-44bd-8889-4edf27210c8b)

I'm not sure if the Jellyfin media player on Linux has this option too, but I hope this helps as a reference!

**Reply by jelabarre59** [reply]
There's a checkbox in the same place, but Jellyfin server v10.9 is on Fedora now.   Granted, now that I'm not at my old job I may be switching back to Linux Mint anyway.
